act/demographics_mapping.sql

```diff
@@ -151,3 +151,22 @@ insert /*+ APPEND */ into nightherondata.concept_dimension (
 commit;
 
```

Comment:
--- land act_covid concepts in concept_dimension ; missleading comment at line 126